fastly/resource_fastly_compute_acl_entries.go

```diff
@@ -0,0 +1,199 @@
+package fastly
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/fastly/go-fastly/v10/fastly/computeacls"
+
+	gofastly "github.com/fastly/go-fastly/v10/fastly"
+)
+
+func resourceFastlyComputeACLEntries() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceFastlyComputeACLEntriesCreate,
+		ReadContext:   resourceFastlyComputeACLEntriesRead,
+		UpdateContext: resourceFastlyComputeACLEntriesUpdate,
+		DeleteContext: resourceFastlyComputeACLEntriesDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceFastlyComputeACLEntriesImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"compute_acl_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Manages entries for a Fastly Compute Access Control List (ACL). To import, use the format <compute_acl_id>/entries.",
+			},
+			"entries": {
+				Type:        schema.TypeMap,
+				Required:    true,
+				Description: "A map representing an entry in the Compute ACL, where the key is the prefix and the value is the action (ALLOW or BLOCK).",
+				Elem:        schema.TypeString,
+				DiffSuppressFunc: func(_, _, _ string, d *schema.ResourceData) bool {
+					return !d.HasChange("compute_acl_id") && !d.Get("manage_entries").(bool)
+				},
+			},
+			"manage_entries": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Have Terraform manage the entries (default: false). If true, Terraform will remove entries not defined in the HCL.",
```

Comment:
This *looks* like a typo (HCL instead of ACL), but it's not, it's correct. It's possible that other readers will think that too, so we can write the sentence differently to avoid the confusion.

Something like "Manage the ACL entries in Terraform (default: false). If true, Terraform will ensure that the ACL's entries match the entries in the Terraform configuration."